app/webpacker/scripts/form-error-tracking.js

```diff
@@ -0,0 +1,7 @@
+export function triggerFormAnalytics(category, action, label) {
```

Comment:
This method looks generic enough to be something like `sendEventToGa`

Reply:
This is consistent with **search-and-compare-ui** but happy to change.